src/docbuilder/rustwide_builder.rs

```diff
@@ -760,12 +760,75 @@ mod tests {
     use crate::test::wrapper;
 
     #[test]
+    #[ignore]
     fn test_build_crate() {
         wrapper(|env| {
+            let crate_ = "log";
+            let version = "0.4.14";
+            let default_target = "x86_64-unknown-linux-gnu";
+            // env.fake_release()
+            //     .name(crate_)
+            //     .version(version)
+            //     .default_target(default_target)
+            //     .add_target("i686-pc-windows-msvc")
+            //     .add_target("i686-unknown-linux-gnu")
+            //     .add_target("x86_64-apple-darwin")
+            //     .add_target("x86_64-pc-windows-msvc")
+            //     .create()?;
+
             let mut builder = RustwideBuilder::init(env).unwrap();
             builder
-                .build_package("log", "0.4.14", PackageKind::CratesIo)
-                .map(|_| ())
+                .build_package(crate_, version, PackageKind::CratesIo)
+                .map(|_| ())?;
+
+            // check release record in the db (default and other targets)
+            let mut conn = env.db().conn();
+            let rows = conn
+                .query(
+                    "SELECT 
+                        r.rustdoc_status,
+                        r.default_target,
+                        r.doc_targets
+                    FROM 
+                        crates as c 
+                        INNER JOIN releases AS r ON c.id = r.crate_id
+                    WHERE 
+                        c.name = $1 AND 
+                        r.version = $2",
+                    &[&crate_, &version],
+                )
+                .unwrap();
+            let row = rows.get(0).unwrap();
+
+            assert_eq!(row.get::<_, bool>("rustdoc_status"), true);
+            assert_eq!(row.get::<_, String>("default_target"), default_target);
+            assert_eq!(
+                row.get::<_, Value>("doc_targets"),
+                json!([
+                    "x86_64-unknown-linux-gnu", // default target is first
+                    "i686-pc-windows-msvc",     // rest is alphabetically
+                    "i686-unknown-linux-gnu",
+                    "x86_64-apple-darwin",
+                    "x86_64-pc-windows-msvc",
+                ])
```

Comment:
Hmm, I don't know if we should guarantee the order. Maybe just sort the targets before adding the assert?

Reply:
The current impl seems to guarantee it, but I will change the test to sort first.

Reply:
actually, the tests failed on this :) So sorting it will be